packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -1799,6 +1800,79 @@ void main() {
       lessThan(600 - windowPaddingBottom), // Device height is 600.
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    Widget buildFrame({ bool? widgetEnableFeedack, bool? themeEnableFeedback }) {
+      return MaterialApp(
+            home: Scaffold(
+              body: PopupMenuTheme(
+                data: PopupMenuThemeData(
+                  enableFeedback: themeEnableFeedback,
+                ),
+                child: PopupMenuButton<dynamic>(
+                  enableFeedback: widgetEnableFeedack,
+                  child: const Text('Show Menu'),
+                  itemBuilder: (BuildContext context) {
+                    return <PopupMenuItem<int>>[
+                      const PopupMenuItem<int>(
+                        value: 1,
+                        child: Text('One'),
+                      ),
+                    ];
+                  },
+                ),
+              ),
+            ),
+          );
+    }
+
+    testWidgets('PopupMenuButton enableFeedback works properly', (WidgetTester tester) async {
+      //PopupMenuButton with enabled feedback
+      await tester.pumpWidget(buildFrame(widgetEnableFeedack: true));
+      await tester.tap(find.byType(PopupMenuButton));
```

Comment:
Use `tester.tap(find.text('Show Menu'))` to avoid the problem with `PopupMenuButton<int>`?

Reply:
Made the changes. Thank you for guiding :).